examples/server/a_bit_of_everything.go

```diff
@@ -83,7 +82,7 @@ func (s *_ABitOfEverythingServer) BulkCreate(stream examples.StreamService_BulkC
 			return err
 		}
 		count++
-		glog.Error(msg)
+		glog.Info(msg)
```

Comment:
This was outputting error level logging during the test suite, I suspect it's left over from somebody's debugging. It doesn't seem to be an error case.

Reply:
Nice find, I think we can leave it in since this is just used for testing and as an example.